README.md

```diff
@@ -8,6 +8,9 @@ The Travel Time Request App is a simple web application designed to help City st
 This app was originally developed as a [class project by U of T students](https://www.youtube.com/watch?v=y6lnefduogo) in partnership with the City, though it has undergone substantial development by the Data & Analytics Unit since then. 
 
 ## How to use the app
+
+### Via the front-end user-interface 
+
 When you [visit the app](https://trans-bdit.intra.prod-toronto.ca/traveltime-request/), you will be prompted to add/create at least one of each of the following:
 * a corridor, drawn on the map
 * a time range, given in hours of the day, 00 - 23
```

Comment:
Are time ranges exclusive of the end of the interval? E.g., is [23, 24) valid?

Reply:
Good question. I should have updated this in https://github.com/CityofToronto/bdit_tt_request_app/pull/160. I'll make a new PR for that.